authenticators/components/org.wso2.carbon.identity.sample.local.authenticator/README.md

```diff
@@ -0,0 +1,81 @@
+# Implement a custom local authenticator for WSO2 Identity Server
+
+The default authenticator in the WSO2 Identity Server is the Basic authenticator. It is a local authenticator that authenticates the end users using a connected user store and the provided user name and password.
+
+WSO2 Identity Server comes with extensibility which allows you to change the authentication logic which the local users are validated. You only have to write a local authenticator with the customized authentication logic and deploy it in the WSO2 Identity Server.
+
+This guide discusses the three main steps required when introducing custom local authenticator to the WSO2 Identity Server and how you can try out the custom authenticator. 
+
+1. Implement the custom local authenticator
+2. Deploy the authenticator in the WSO2 IS 
+3. Configure an application with the custom authenticator
+4. Trying out the custom authenticator
+
+## Sample scenario
+
+Let’s say we need to login into an application using our telephone number(http://wso2.org/claims/telephone) instead of the username. Therefore if the user enters his/her mobile number as the username, the authentication logic should validate the credentials of the user identified by the given telephone number.
```

Comment:
```suggestion
Let’s say we need to log in to an application using our telephone number(http://wso2.org/claims/telephone) instead of the username. Therefore if the user enters his/her mobile number as the username, the authentication logic should validate the credentials of the user identified by the given telephone number.
```